packages/components/package.json

```diff
@@ -11,7 +11,7 @@
   "bugs": {
     "url": "https://github.com/goodhood-eu/goodhood/issues"
   },
-  "version": "6.2.0",
+  "version": "6.3.0-beta.0",
```

Comment:
do we really want to merge with beta version?

Reply:
no, will update to release version